spec/models/topic_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+describe Topic do
+  describe 'image' do
+    let(:topic) { create(:topic, :with_avatar) }
+    it 'have avatar attached' do
+      expect(topic.avatar.attached?).to be
+    end
+  end
```

Comment:
You should also add tests for the model validations, like `presence: true` or `uniqueness: true`
Check this gem that might be useful:
https://github.com/thoughtbot/shoulda-matchers